lib/Query_Processor.cpp

```diff
@@ -431,7 +431,7 @@ static void __delete_query_rule(QP_rule_t *qr) {
 // delete all the query rules in a Query Processor Table
 // Note that this function is called by GloQPro with &rules (generic table)
 //     and is called by each mysql thread with _thr_SQP_rules (per thread table)
```

Comment:
This comment needs correction. 
Now `__reset_rules()` is also called from Admin .
We also need to describe why

Reply:
Now properly commented.